pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/instance/DDPInstanceDao.java

```diff
@@ -264,6 +265,30 @@ public Optional<DDPInstanceDto> getDDPInstanceByInstanceName(@NonNull String ins
         return Optional.ofNullable((DDPInstanceDto) results.resultValue);
     }
 
+    public Optional<DDPInstanceDto> getDDPInstanceByStudyGuid(@NonNull String studyGuid) {
```

Comment:
This error was missing in DDPInstanceDao, basically this is to be able to select the ddpInstaceDto based on the stuyd-guid